### ISSUE TITLE
Incubus

### ISSUE THREAD

**Issue by Lonrite** [UnderscoreKilburn/repentance-issues#1483]
sometimes glitches out for no reason and his wings stop flapping even when you keep going other places

**Comment by ghost** [comment]
Yeah I've noticed this too, in my experience this happens most often when you also have C-section for some reason.

**Comment by ZMBanshee** [comment]
Yes, I've also seen this with C Section. Post another comment here if there are other items that cause that.

**Comment by Lonrite** [comment]
yeah it seems like C Section is the main case (maybe even the only one but idr/idk 4sho)
